test/conftest.py

```diff
@@ -196,17 +195,7 @@ def infer_required_doc_store(item, keywords):
     # 2. if the test name contains the docstore name, we use that
     # 3. use an arbitrary one by calling set.pop()
     required_doc_store = None
-    all_doc_stores = {
-        "elasticsearch",
-        "faiss",
-        "sql",
-        "memory",
-        "milvus1",
-        "milvus",
-        "weaviate",
-        "pinecone",
-        "opensearch",
-    }
+    all_doc_stores = {"elasticsearch", "faiss", "sql", "memory", "milvus1", "milvus", "weaviate", "pinecone"}
```

Comment:
So as we add test_memory.py, it will get removed from these existing "references"? We'll do that one-by-one for each document store @masci ?

Reply:
@vblagoje correct, up to the point where we can remove `test_document_store.py`